main.tf

```
@@ -89,22 +89,107 @@ module "kyverno" {
   admission_controller = var.kyverno.admission_controller
 }
 
-module "alloy" {
-  count = var.alloy.enabled ? 1 : 0
-
-  source = "./modules/alloy"
-
-  namespace                      = local.namespace
-  loki                           = var.alloy.loki
-  loki_scrape_global             = var.alloy.loki_scrape_global
-  loki_collect_self_logs_enabled = var.alloy.loki_collect_self_logs_enabled
-  prometheus                     = var.alloy.prometheus
-  tenant_id                      = var.alloy.tenant_id
-  node_template_file_path        = var.alloy.node_template_file_path
-  cluster_template_file_path     = var.alloy.cluster_template_file_path
-
-  cluster_name = var.cluster_name
-  project      = var.project
+module "fluent-bit" {
+  count = var.fluent_bit.enabled ? 1 : 0
+
+  source = "./modules/fluent-bit"
+
+  namespace    = local.namespace
+  logs_storage = var.fluent_bit.logs_storage
+  loki = {
+    tenant_id = var.fluent_bit.loki.tenant_id
+    basic_auth = {
+      enabled  = var.fluent_bit.loki.basic_auth.enabled
+      username = var.fluent_bit.loki.basic_auth.username
+      password = var.fluent_bit.loki.basic_auth.password
+    }
+    bearer_token = {
+      enabled = var.fluent_bit.loki.bearer_token.enabled
+      token   = var.fluent_bit.loki.bearer_token.token
+    }
+  }
+  elasticsearch = {
+    auth = {
+      enabled  = var.fluent_bit.elasticsearch.auth.enabled
+      username = var.fluent_bit.elasticsearch.auth.username
+      password = var.fluent_bit.elasticsearch.auth.password
+    }
+  }
+  use_defaults = {
+    outputs = var.fluent_bit.use_defaults.outputs
+    filters = var.fluent_bit.use_defaults.filters
+    inputs  = var.fluent_bit.use_defaults.inputs
+  }
+  logs_custom = {
+    outputs = var.fluent_bit.logs_custom.outputs
+    filters = var.fluent_bit.logs_custom.filters
+    inputs  = var.fluent_bit.logs_custom.inputs
+  }
+  logs_endpoint_url = var.fluent_bit.logs_endpoint_url
+
+
+}
+
+module "grafana_alloy_cluster" {
+  count = var.grafana_alloy.cluster.enabled ? 1 : 0
+
+  source                  = "cookielab/grafana-alloy/kubernetes//modules/cluster"
+  version                 = "0.0.1"
```

Is the module published? I was unable to find it in the registry. Also the tags prior to the 0.0.2 are prefix with `v`
```suggestion
  version                 = "0.0.2"
```